common/Cargo.toml

```diff
@@ -20,6 +20,7 @@ hyper.workspace = true
 ipnetwork.workspace = true
 macaddr.workspace = true
 lazy_static.workspace = true
+oximeter.workspace = true
```

Comment:
Would it be better to reverse this? Namely, to have `oximeter` depend on `common`, rather than the other way around?

(This current construction seems like it prevents oximeter from depending on common in the future, which it might reasonably want to do)

Reply:
I wondered the same thing. I'll swap it around, thanks.